### ISSUE TITLE
Join the Xamarin Forms Toolkit

### ISSUE THREAD

**Issue by danielmeza** [YauGoh/Xamarin.Responsive#1]
@YauGoh have you thought of join this control to the Xamarin Community Toolkit?

**Comment by YauGoh** [comment]
Yes, that that is something I have thought of contributing to.  Though this is a direct rip off of the responsive grid from an older version of bootstrap, I'm not sure how this would be received.

**Comment by danielmeza** [comment]
We should try, grid system is a common feature in many html frameworks, and it's useful for web developers that want to use their skills on Xamarin. Besides this work well!

**Comment by YauGoh** [comment]
You are right. Might start putting a pull request together this weekend.

**Comment by bricefriha** [comment]
Any news about this? 😁

**Comment by YauGoh** [comment]
I've put my effort to get everything ready for MAUI.

**Comment by AmSmart** [comment]
Hi @YauGoh, Any progress on the plans to set this up for MAUI. I can help contribute if need be.

**Comment by YauGoh** [comment]
Hi,
I've started a new repo that is a port to Maui. Currently it is based on a
earlier version was hoping to look into it once things settle down. There
was a major reactor in the Maui lauot logic that change my implementation a
few times.

**Comment by AmSmart** [comment]
Ahh, I see. Glad to hear. I believe that would be [My.Maui.Responsive](https://github.com/YauGoh/My.Maui.Responsive) so I can keep an eye on it
